lib/expandable.dart

```diff
@@ -163,6 +174,20 @@ class _ExpandablePanelState extends State<ExpandablePanel> {
 
   @override
   Widget build(BuildContext context) {
+    CrossAxisAlignment headerAlignment;
+
+    switch (widget.headerAlignment){
+      case ExpandablePanelHeaderAlignment.center:
+        headerAlignment = CrossAxisAlignment.center;
+        break;
+      case ExpandablePanelHeaderAlignment.bottom:
+        headerAlignment = CrossAxisAlignment.end;
+        break;
+      case ExpandablePanelHeaderAlignment.top:
+      default:
+        headerAlignment = CrossAxisAlignment.start;
+    }
+
```

Comment:
You forgot about icon alignment (line 200). Perhaps we don't need the `Align` widget at all? Try removing it and see the result.

Reply:
It works quite well without the Align widget. I've removed it